plugins/leemons-plugin-layout/frontend/src/hooks/useIsOverHeader.js

```diff
@@ -0,0 +1,6 @@
+import { useMouse } from '@mantine/hooks';
+
+export default function useIsOverHeader() {
+  const { y, x } = useMouse();
+  return y < 80 && x > 0 && y > 0;
```

Comment:
Due to the Impersonation header having the interactions disabled, be can't get the isHover, so we need to use this approach.

The x > 0 && y > 0 is due to the initial value and coords resetting when going outside of the viewport